index.html

```diff
@@ -2347,8 +2347,7 @@ <h4 class="modal-title">Export video</h4>
                             <div class="col-sm-9">
                                 <p class="form-control-static">
                                     If you experience problems with the background flight video being glitchy in the exported video,
-                                    <a
-                                        target="_blank" href="https://github.com/cleanflight/blackbox-log-viewer/blob/master/Readme.md">follow the instructions here</a> to re-encode your flight video.
+                                    <a href="https://github.com/cleanflight/blackbox-log-viewer/blob/master/Readme.md" target="_blank" rel="noopener noreferrer">follow the instructions here</a> to re-encode your flight video.
```

Comment:
Should be https://github.com/betaflight/blackbox-log-viewer/blob/master/README.md.

Reply:
Fixed, with others references to the same link in the file.